app/src/main/java/com/example/githubusers/service/GithubAPI.java

```diff
@@ -7,6 +7,6 @@
 
 
 public interface GithubAPI {
-    @GET("/search/users?q=+language:java+location:nairobi&per_page=100")
+    @GET("/search/users?q=+language:java+location:moscow&per_page=100")
     Call<GithubUsersResponse> getUsers();
```

Comment:
Nairobi is the specified location query parameter, kindly fix that 😄

Reply:
On it